src/js/popup.js

```diff
@@ -51,11 +51,39 @@ function showSignUpPanel() {
 }
 
 
-function choosePanel(numRemaining, panelId, premium){
-  if (premium){
+function premiumFeaturesAvailable(premiumEnabledString) {
+  if (premiumEnabledString === "True") {
+    return Boolean(true);
```

Comment:
Since you're using `return` in the `if`, you can drop the `else`:

```js
  if (premiumEnabledString === "True") {
    return true;
  }
  return false;
```

Or one-liner it:

```js
  return premiumEnabledString === "True";
```